backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -86,4 +92,15 @@ public void updateUserById(
 		}
 	}
 
+	@PostMapping
+	@ResponseStatus(HttpStatus.NO_CONTENT)
+	@Operation(summary = "Create a new user")
+	public void createUserByEmailAndUserAttributes(@Validated @RequestBody UserCreateModel userCreateModel) 
+	{
+		UserModel userModel = new UserModel();
+		userModel.setUserAttributes(userCreateModel.getUserAttributes());
+		userModel.setEmail(userCreateModel.getEmail());
+		
+		userService.createUser(userModelMapper.toDomainObject(userModel));
```

Comment:
The mapper should map from `UserCreateModel` (not `UserModel)` to `User`. That way, this method won't have to map from `UserCreateModel` --> `UserModel` --> `User`

Reply:
new model created